boxnav/boxsim.py

```diff
@@ -6,6 +6,8 @@
 from time import sleep
 
 import enlighten
+from subprocess import run as sprun
+from os import chdir
```

Comment:
Are these being used in boxsim?

Reply:
Yes they are, they may have been duplicated during the merge.